src/main/java/gov/cdc/sdp/cbr/queue/DatabaseQueueEndpoint.java

```diff
@@ -26,95 +26,107 @@
 
 @UriEndpoint(scheme = "sdpqueue", title = "SDPQueue", syntax = "sdpqueue", label = "database,sql,queue")
 public class DatabaseQueueEndpoint extends DefaultEndpoint {
-    private static final Logger LOG = LoggerFactory.getLogger(DatabaseQueueEndpoint.class);
```

Comment:
Changes in this class revert from space-based indent to tabs.  Please update eclipse format settings to do 4 spaces and resubmit.